### ISSUE TITLE
Installed apps are 'Paused'

### ISSUE THREAD

**Issue by robot5x** [holochain/launcher-tauri#38]
Hi -

Launcher starts fine, and apps can be installed successfully. However they always appear immediately as 'Paused' and this cannot be changed. The info message says:
`Some cells are missing / not able to run: [ CellId( DnaHash(uhC0kDj9yRoE7c87I_c6Ch2U8EPKxJF-Hqk9dfyYDpnagTPgPRnFH), AgentPubKey(uhCAk6Knva4eQb4Gn8HIhxls9atGn8Y9Mu1XiMMHkO-nivbRiTVeK), ), ]`

Log of last run attached.
[launcher2.log](https://github.com/holochain/launcher/files/7819282/launcher2.log)

This laptop is identical to one I can run apps successfully on, except no holochain tools are already installed. Machine is archlinux running kernel 5.15.12, installed via AUR.

The webHapp is built with holochain 0.0.115, but I confirm the issue occurred on previous launcher version 0.3.6 (I upgraded now to 0.3.7 to see if it would fix, but unfortunately not).

Thanks - alex

**Comment by abe-njama** [comment]
Thanks for raising this. We've investigated this and are more drawn to the Conductor not using the [Now](https://github.com/holochain/bootstrap#now) service in the Bootstrap API. This is being looked into further.

**Comment by joatu** [comment]
I am currently experiencing this issue with the 'Where' app. It worked until I decided to install the latest version of Holochain Launcher 38, from 37. I hadn't uninstalled the old one first. I couldn't get the 'where' app to load.

I restarted, it didn't change anything. I uninstalled and reinstalled. Once the new install came alive, all of the apps were still there (which was a surprise, is that a bug or a feature?). And now the 'where' app is stuck on Pause.

![image](https://user-images.githubusercontent.com/5350076/152853172-1bba5a16-6df0-45c9-9ea2-65b49137f9ab.png)
